content/en/docs/tasks/security/authentication/authn-policy/index.md

```diff
@@ -506,13 +529,13 @@ EOF
 Retry the request without a token. The request now fails with error code `403`:
 
 {{< text bash >}}
-$ curl $INGRESS_HOST/headers -s -o /dev/null -w "%{http_code}\n"
+$ curl "$INGRESS_HOST:$INGRESS_PORT"/headers -s -o /dev/null -w "%{http_code}\n"
```

Comment:
This better, but a common convention is to wrap the entire single bash arg in quotes. Also, then you don't need to show quotes when describe it in the text (see my other suggestion, below).
```suggestion
$ curl "$INGRESS_HOST:$INGRESS_PORT/headers" -s -o /dev/null -w "%{http_code}\n"
```
Also, don't forget to regen the snips.

Reply:
That's better.  Should be fixed in the new version.